services/IDonor.cs

```diff
@@ -0,0 +1,37 @@
+using Stateless;
+
+namespace NosAyudamos
+{
+    public interface IDonor
+    {
+        State State { get; set; }
+        void Register();
+    }
+
+    public class Donor : IDonor
+    {
+        private readonly StateMachine<State, Trigger> _machine;
+
+        public State State { get; set; }
+
+        public Donor()
+        {
+            _machine = new StateMachine<State, Trigger>(() => State, s => State = s);
+            InitializeStateMachine();
+        }
+
+        public void Register()
```

Comment:
I like one-liner members to just use lambdas, like `public void Register() => _machine.Fire(Trigger.Register);`

Reply:
will not be on line member, implementation is not complete